tests/test_bookmarks.py

```diff
@@ -0,0 +1,150 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Tests for Bookmarks."""
+
+import pytest
+from selenium.webdriver.remote.webdriver import WebDriver
+from foxpuppet.windows import BrowserWindow
+from foxpuppet.windows.browser.bookmarks.bookmark import (
+    Bookmark,
+    BookmarkData,
+)
+
+
+class BookmarkEntry:
+    """Class representing a Firefox bookmark entry."""
+
+    def __init__(self, url: str, name: str = "", keyword: str = "", tags: list[str] = []):
+        self.url = url
+        self.name = name
+        self._keyword = keyword
+        self._tags = tags or []
+
+    @property
+    def keyword(self):
+        """Returns the current list of keywords."""
+        return self._keyword
```

Comment:
Is this a list or just 1 word, it is currently set as a `str`.

Reply:
I will correct that clarity mistake.